### ISSUE TITLE
it wont click why

### ISSUE THREAD

**Issue by SergioCupido** [MOj0/LoR-Bot#38]
![image](https://github.com/MOj0/LoR-Bot/assets/97339303/9c9e30d7-1ce4-4a5a-b751-4ccd7b1f7a8d)

**Comment by SergioCupido** [comment]
it say opponent_turn but its my turn to end round why it doesnt click

**Comment by MOj0** [comment]
Right, this is a limitation (not sure if it is mentioned in the readme).
The issue is that bot does not get any info regarding the "turn status".
The solution is simple: enable auto-pass in game settings.
